setzer/dialogs/interpreter_missing/interpreter_missing.py

```diff
@@ -44,7 +44,16 @@ def setup(self, interpreter_name):
         self.view = Gtk.MessageDialog(self.main_window, 0, Gtk.MessageType.QUESTION)
 
         self.view.set_property('text', _('LateX Interpreter is missing.'))
-        self.view.format_secondary_markup(_('''Setzer is configured to use »{interpreter}« which seems to be missing on this system.
+        if 'FLATPAK_ID' in os.environ:
```

Comment:
To check whether it's running as Flatpak it's best to use [Xdp](https://lazka.github.io/pgi-docs/#Xdp-1.0) (https://lazka.github.io/pgi-docs/#Xdp-1.0/classes/Portal.html#Xdp.Portal.running_under_flatpak), although that would make it depend on libportal 😅